releasenotes/notes/parallel_passmanager_run-1d2173d33854e288.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    When ``passmanager.run(...)`` is invoked with more than one circuit, the transpilation of
+    these circuits will run in parallel. Because of this upgrade, the module ``dill`` is a
+    Terra requirement now.
```

Comment:
Can we reword this a be to be something like:
```
[dill](https://pypi.org/project/dill/) was added as a requirement. This is needed to enable running
``passmanager.run()`` in parallel for more than one circuit.
```

It just feels like the key bit here in the upgrade note is that the requirement is new so we should say that first.

Reply:
Maybe both?